### PR TITLE
implement showcases section

### PR DESCRIPTION
**Pull request by erizocosmico**
screenshot_2017-06-09_12 47 38

As you can see in my screenshot with a screen of 1280px of width the pills of the titles don't fit.
The black lines on the left are because of the screenshot (are the borders of my chrome window)

### Some explanations

* Why is the h2 title of the project outside the slider? Because slick needs a `overflow: hidden` to work and thus the pill does not show.
* Why does the "check this project" button have a left margin of 5px? Same reason as above but with the hover shadow (it can be fixed with CSS but the effect is gonna be the same and it has the drawback of being displayed everywhere as opposed to just here).

### Questions

* @Serabe does it need to have autoplay now that it's not a proper slider?

### CODE REVIEW

File: hugo/layouts/partials/home/showcases.html [hunk 1]
```diff
@@ -1,15 +1,45 @@
-<h1>Showcases</h1>
-{{ range $p := $.Site.Data.home.showcases.showcases }}
-    <div>
-        <h2>{{$p.title}}</h2>
-        <p>{{$p.desc}}</p>
-        <a href="{{ $p.link }}" target="_blank">check this project</a>
+<div class="showcases">
+    <div class="showcases__info">
+        <h1>Showcases</h1>
```
Comment (Serabe) [inline hugo/layouts/partials/home/showcases.html]: For this `h1` you need an element that resets the headers, like `article` or `section`.

File: hugo/layouts/partials/home/showcases.html [hunk 2]
```diff
@@ -1,15 +1,45 @@
...
+            <div>
+                <p>{{ $s.desc }}</p>
+
+                <a href="{{ $s.link }}" target="_blank" title="{{ $s.title }}" class="btn-pill">check this project</a>
+            </div>
+            {{ end }}
+        </div>
+    </div>
+
+    {{ range $s := (first 1 .showcases) }}
```
Comment (Serabe) [inline hugo/layouts/partials/home/showcases.html]: Either remove these parentheses or add them to :5

**Comment by erizocosmico** [comment]
Done @Serabe

File: hugo/layouts/partials/home/showcases.html [hunk 3]
```diff
@@ -1,15 +1,45 @@
...
-    <div>
-        <h2>{{$p.title}}</h2>
-        <p>{{$p.desc}}</p>
-        <a href="{{ $p.link }}" target="_blank">check this project</a>
+<section class="showcases">
+    <div class="showcases__info">
+        <h1>Showcases</h1>
+        
+        {{ range $s := first 1 .showcases }}
+        <h2 id="showcase-title">{{ $s.title }}</h2>
```
Comment (dpordomingo) [inline hugo/layouts/partials/home/showcases.html]: pretty please, `tab` after a `range` :)

File: hugo/layouts/partials/home/showcases.html [hunk 4]
```diff
@@ -1,15 +1,45 @@
...
+                <a href="{{ $s.link }}" target="_blank" title="{{ $s.title }}" class="btn-pill">check this project</a>
+            </div>
+            {{ end }}
+        </div>
+    </div>
+
+    {{ range $s := first 1 .showcases }}
+    <div class="showcases__terminal">
+        <div class="showcases__terminal__header">
+            <div></div>
```
Comment (dpordomingo) [inline hugo/layouts/partials/home/showcases.html]: I love it ;)

File: hugo/layouts/partials/home/showcases.html [hunk 5]
```diff
@@ -1,15 +1,45 @@
...
+            <div></div>
+            <div></div>
+        </div>
+
+        <div id="showcase-code" class="showcases__terminal__code">
```
Comment (dpordomingo) [inline hugo/layouts/partials/home/showcases.html]: missing `tab`

**Pull request merged**